app/components/ProgressBar/index.tsx

```diff
@@ -51,6 +51,8 @@ function ProgressBar(props: Props) {
         subVariable,
     } = props;
 
+    console.warn('total', totalValue);
```

Comment:
Please remove this console or keep a NOTE.

Reply:
Oooops!